packages/api-client/src/APIClient.ts

```diff
@@ -200,7 +200,7 @@ export class APIClient extends EventEmitter {
   }
 
   private configureApis(backendFeatures: BackendFeatures): Apis {
-    this.logger.info('configuring APIs with config', backendFeatures);
+    this.logger.info('configuring APIs with config', JSON.stringify(backendFeatures));
```

Comment:
after changes in https://github.com/wireapp/wire-webapp/pull/18359, shouldn't it be done automatically?

Reply:
web-packages use a separate instance of logdown of their own

Reply:
see https://github.com/search?q=repo%3Awireapp%2Fwire-web-packages%20%3D%20logdown(&type=code

Reply:
okay, so maybe we should copy-paste the solution from the webapp you have provided? then we wouldn't have to stringify those args in the future, WDYT?

Reply:
doing a big refactor

Reply:
https://github.com/wireapp/wire-web-packages/pull/6696/commits/b25192005674d1fb61b74012850a9fc0fa9a4030